File: src/iniciante/Bee1015.java
package iniciante;

import java.util.Scanner;

public class Bee1015 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        float x1 = input.nextFloat();
        float y1 = input.nextFloat();
        float x2 = input.nextFloat();
        float y2 = input.nextFloat();

        float diferencaX = ((x2 - x1) * (x2 - x1));
        float diferancaY = ((y2 - y1) * (y2-y1));
        float diferenca = diferencaX + diferancaY;

        //double distancia = Math.sqrt(Math.pow(diferencaX, diferancaY) - Math.pow(diferancaY, diferancaY));
        double distancia = Math.sqrt(diferenca);

        System.out.println(String.format("%.4f", distancia));

    }
}

File: beginner/bee1015.py
from math import sqrt

def distance(x, y, x2, y2):
    diference_x = x2 - x
    diference_y = y2 - y
    distance = sqrt((pow(diference_x, 2) + pow(diference_y, 2)))
    return distance

x, y = input().split()
x = float(x)
y = float(y)

x2, y2 = input().split()
x2 = float(x2)
y2 = float(y2)

print(f'{distance(x, y, x2, y2):.4f}')
